Apps/RaptureIntTests/build.gradle

```diff
@@ -77,3 +73,18 @@ task docTests (type: Test) {
     }
 
 }
+
+task pythonSearchTests(type: Exec) {
+    description 'Run Python search tests using nosetests'
+    println 'Running Python search tests'
+    commandLine nosetestsCmd, 'src/test/python/searchTests.py'
+}
+
+task pythonTests(type: Test, dependsOn: pythonSearchTests) {
+    description 'Run Python tests'
+    println 'Running Python tests'
+}
+
+// This doesn't seem to work
+pythonTests.mustRunAfter nightlyTests
```

Comment:
As a reminder that it doesn't work, in the hope that either I or someone else will figure out what the problem is. It doesn't cause a problem, but it's supposed to have the Python tests run after the Java tests - and that part isn't happening.